db/schema.sql

```diff
@@ -156,7 +156,8 @@ CREATE TABLE batch_unit_aggregations(
 CREATE TYPE COLLECT_JOB_STATE AS ENUM(
     'START',     -- the aggregator is waiting to run this collect job
     'FINISHED',  -- this collect job has run successfully and is ready for collection
-    'ABANDONED'  -- this collect job has been abandoned & will never be run again
+    'ABANDONED', -- this collect job has been abandoned & will never be run again
+    'DELETED'    -- this collect job has been deleted due to an explicit request by the collector
```

Comment:
I think this "deleted" state should be used for GC'ed collect jobs, as well, once we implement GC. WDYT?

If you agree, I think this comment should be updated, either to remove the "due to an explicit request by the collector" or to make it clear that's just one way that a collect job can be deleted.